### DISCUSSION TITLE
Rofi dmenu theme

### DISCUSSION THREAD

**Discussion by mmarco2121** [davatorium/rofi#1241] [Q&A] [answered]
Is it possible to edit the position of rofi in the dmenu rofi theme? 

The anchor and location are set to north. I'd like to be able to pull it a few pixels down from the very top edge of the screen, but cannot figure out a way to do so. Any help would be appreciated.

**Comment by DaveDavenport** [comment, marked as answer]
try:
```css
window {
   x-offset: 12;
   y-offset: 12;
}
```
in your configuration file after importing the dmenu theme.

(See manpage for these properties).

**Reply by mmarco2121** [reply]
Thank you. This helped.

**Reply by DaveDavenport** [reply]
Great!
